server/data/runeTrees.json

```diff
@@ -0,0 +1,387 @@
+[
```

Comment:
I copied this file to the server, too, so if knows which runes are in which trees